### DISCUSSION TITLE
What impacts if keda-operator-metrics-apiserver pod "Failed to make webhook authorizer request"?

### DISCUSSION THREAD

**Discussion by fanyangMicrosoft** [kedacore/keda#3979] [Q&A / Need Help]
Hi, I checked the logs of keda-operator-metrics-apiserver pod (by command: kubectl logs <keda-operator-metrics-apiserver-pod-name>) and got the following error in the logs: Failed to make webhook authorizer request: Post "[https://10.3.0.1:443/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s](https://10.3.0.1/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s)[":](https://10.3.0.1/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s%22:) dial tcp 10.3.0.1:443: connect: connection refused
E1129 20:49:01.569881      1 errors.go:77] Post "[https://10.3.0.1:443/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s](https://10.3.0.1/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s)[":](https://10.3.0.1/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s%22:) dial tcp 10.3.0.1:443: connect: connection refused. 

May I know what may cause this connection failure and what impacts if there exists this connection failure?

In the keda-operator-metrics-apiserver pod, aadpodidbinding: dev-fist-keda-pod-identity.  What roleAssignment should be assigned to dev-fist-keda-pod-identity to allow it to make webhook authorizer request?

My confusion is what resource that metics server is trying to connect? what roleAssignment access should be assigned to the identity?

**Comment by JorTurFer** [comment]
Sorry for the late response, I didn't see the discussion.
I think this is an internal error from the custom metrics server that it's self-recover
